assets/css/gitshame.css

```diff
@@ -1,9 +1,62 @@
+:root {
+  /*https://color.adobe.com/creamy-grey-color-theme-7775*/
```

Comment:
Yeah, dealing with assets is a bit of a sore point in go right now... A couple partial solutions:
- https://github.com/mijia/gobuildweb
- https://github.com/shaoshing/train

I've used train before, I can help you set it up if you want. It does the job, but it's not perfect :/